doc/userguide/configuration/suricata-yaml.rst

```diff
@@ -337,6 +337,38 @@ integration with 3rd party tools like logstash.
             # the example below adds three additional fields when uncommented
             #custom: [Accept-Encoding, Accept-Language, Authorization]
         - dns:
+            # Use version 2 logging with the new format:
+            # dns answers will be logged in one single event
+            # rather than an event for each of it.
+            # Without setting a version the version
+            # will fallback to 1 for backwards compatibility.
+            version: 2
+
+            # Enable/disable this logger. Default: enabled.
+            #enabled: no
+
+            # Control logging of requests and responses:
+            # - requests: enable logging of DNS queries
+            # - responses: enable logging of DNS answers
+            # By default both requests and responses are logged.
+            #requests: no
+            #responses: no
+
+            # Format of answer logging:
+            # - detailed: array item per answer
+            # - grouped: answers aggregated by type
+            # Default: all
+            #answer-format: [detailed, grouped]
+
+            # Answer types to log.
+            # Default: all
+            #answer-types: [a, aaaa, cname, mx, ns, ptr, txt]
```

Comment:
I know I suggested this name, but I realize in git master this applies to the requests that are logged as well. So I question the "answer-types" name.

I wonder if a name like "types" or "rrtypes" (but not "custom") is better here as it applies to requests and responses.

Reply:
I'd say we rename `answer-types` to `types` and `asnwer-format` to `format`.

Reply:
Works for me.